package.json

```diff
@@ -23,6 +23,10 @@
         "npm-run-all": "^4.1.5"
     },
     "dependencies": {
+        "@emotion/react": "^11.13.3",
```

Comment:
Do these need to be installed at the top level, as opposed to within the client? And, if not, should the versions of these packages match what is in the client's package.json? Why two separate versions?

Reply:
Moved json-viewer into client package, removed instances in top level package.